docs/api/Server.md

```diff
@@ -57,7 +57,7 @@ Authentication APIs are available by default on `WebSocket port` + 1.
 
 ### Creating a game
 
-#### `/games/:name/create`
```

Comment:
What's the benefit of dropping `create` here?

Reply:
That's how [Sir Timothy John Berners-Lee](https://en.wikipedia.org/wiki/Tim_Berners-Lee) wanted it.
URLs repesent resources (games collection in this case) and POST already means 'create'. See also: https://restfulapi.net/http-methods/#post,

(Not very critical in this case though)